panoply/error_normalization.py

```diff
@@ -0,0 +1,97 @@
+from panoply import DataSource, PanoplyException
+from datetime import datetime
+
+
+ERROR_CODES_REGISTRY = {
+    400: 'Bad request',
+    401: 'Unauthorized',
+    403: 'Permissions error',
+    404: 'Resource is not found',
+    408: 'Timeout',
+    422: 'Parsing error',
+    429: 'Rate limit',
+    500: 'Internal exception',
+    501: 'Unclassified error',
+    506: 'SDK error in processing an error'
+}
+
+EXCEPTIONS_REGISTRY = {}
+
+
+class NormalizedException(PanoplyException):
```

Comment:
Normalization is happening inside but that's kinda hiding what this is supposed to be. I suggest we follow something with more meaning.

```suggestion
class DataSourceException(PanoplyException):
```